python-test-samples/apigw-sqs-lambda-sqs/template.yaml

```diff
@@ -132,6 +94,44 @@ Resources:
             BatchSize: 10
       MemorySize: 128
       Timeout: 5
+  WriteTestResult:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: src/write-test-result
+      Handler: write_test_result.lambda_handler
+      Runtime: python3.9
+      MemorySize: 128
+      Timeout: 5
+      Tracing: Active
+      Environment:
+        Variables:
+          TESTRESULTSTABLE_TABLE_NAME: !Ref TestResultsTable
+          TESTRESULTSTABLE_TABLE_ARN: !GetAtt TestResultsTable.Arn
+      Policies:
+        - DynamoDBCrudPolicy:
```

Comment:
Use [DynamoDBWritePolicy] instead of DynamoDBCrudPolicy for least priv.

Reply:
Done